pkgs/by-name/vi/vinegar/package.nix

```diff
@@ -8,22 +8,16 @@
 , xorg
 , wayland
 , vulkan-headers
-, wineWowPackages
-, fetchpatch
+, wine64Packages
 }:
 let
-  # wine-staging doesn't support overrideAttrs for now
-  wine = wineWowPackages.stagingFull.overrideDerivation (oldAttrs: {
+  wine = wine64Packages.stable.overrideDerivation (oldAttrs: {
```

Comment:
Why is wine64 being used instead of wineWow? wineWow provides 32 and 64 bit support whilst wine64 only provides support for 64 bit. Certain bits of Roblox still require 32 bit.

Also, the [vinegar wiki](https://vinegarhq.org/Home/faq.html#what-is-the-best-configuration) recommends using wine staging due to certain patches and features it provides.

Reply:
Okay, I've switched the wine version to staging since it has the required fixes and the patches from the flatpak can properly apply.

Reply:
is there a resolved clarification about switch to `wine64` ?

Reply:
Vinegar now requires the ``wine64`` unlike before which wine64 now provides